APKBUILD

```diff
@@ -1,26 +1,23 @@
 # Maintainer: David Ostrovsky <david@ostrovsky.org>
 
 pkgname=bazel
-pkgver=0.13.0
+pkgver=0.15.2
 pkgrel=0
 pkgdesc='Correct, reproducible, and fast builds for everyone'
 arch="all"
 license="ASL-2.0"
 url='https://bazel.io/'
-depends="openjdk8 libarchive zip unzip"
-makedepends="git protobuf python"
+depends="bash openjdk8 libarchive zip unzip"
+makedepends="coreutils git linux-headers protobuf python"
 options="!distcc !strip"
 source="https://github.com/bazelbuild/bazel/releases/download/${pkgver}/bazel-${pkgver}-dist.zip
         https://github.com/bazelbuild/bazel/releases/download/${pkgver}/bazel-${pkgver}-dist.zip.sig"
 
-sha512sums="3c128e551cff1b685250a68892ca3e1ad6be8b152ee2b8eb527c94adbb8fd50c70e703a363bb938916275030ddb14d756c14e4dc238e7a7b40289c700c5d53c7  bazel-0.13.0-dist.zip
-37fd1446e4c9614d66d85d06a9906b2c9bf3026c6f843af913134abff5d7479864813e29a00b8e6875938994022e3736f9d495f360d7fa5e2e2c4db48a2a28f7  bazel-0.13.0-dist.zip.sig"
+sha512sums="c1257fc7eeadc8e76fbd52be393d07161e6eadf7111e09ee9a7f9bf6d3b7e848e9b193a3b66a13e99c09a42e577404cb3a5f1fc3819f35436a36149dbb1ea365  bazel-0.15.2-dist.zip
+d81fd679b9efbd334a32ed4d32561deed7aace24f12462a5860d3fc7b4fc2dfa3614a23f4772e54d06031b364790e91df936d34b1f881ed4067234a8d84cc0b9  bazel-0.15.2-dist.zip.sig"
 
 build() {
   ./compile.sh
-  # Patch bazel: https://github.com/bazelbuild/bazel/issues/4055
-  # https://bugs.alpinelinux.org/issues/8121
```

Comment:
Can you move this comment to the line where you added coreutils as work around? And say, that this dependency could be elminated when expr in busybox is compatible with  BSD version (or similar).

Reply:
Sure thing. Great suggestion.

Reply:
I got round to doing that.